<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ContactsPage</class>
 <widget class="QWidget" name="ContactsPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>870</width>
    <height>700</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="viewFriendWidget" native="true">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>100</y>
     <width>500</width>
     <height>400</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(225, 255, 202);</string>
   </property>
   <widget class="QPushButton" name="sendMsgButton">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>320</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 16pt &quot;Microsoft YaHei UI&quot;;</string>
    </property>
    <property name="text">
     <string>发送消息</string>
    </property>
   </widget>
   <widget class="QLabel" name="profileLabel">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>80</y>
      <width>100</width>
      <height>100</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="nameLabel">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>150</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="introLabel">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>190</y>
      <width>261</width>
      <height>111</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="deleteFriendButton">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>320</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 16pt &quot;Microsoft YaHei UI&quot;;</string>
    </property>
    <property name="text">
     <string>删除好友</string>
    </property>
   </widget>
   <widget class="QLabel" name="IDLabel">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>100</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="background">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>870</width>
     <height>700</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(244, 255, 225);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QWidget" name="addFriendWidget" native="true">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>100</y>
     <width>500</width>
     <height>400</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(225, 255, 202);</string>
   </property>
  </widget>
  <zorder>background</zorder>
  <zorder>addFriendWidget</zorder>
  <zorder>viewFriendWidget</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
